dags/parquet_export.py

```diff
@@ -34,44 +35,7 @@
 
 # Make sure all the data for the given day has arrived before running.
 # Running at 1am should suffice.
-dag = DAG('main_summary', default_args=default_args, schedule_interval='0 1 * * *', max_active_runs=10)
-
-# We copy yesterday's main pings from telemetry_live to telemetry_stable
-# at the root of this DAG because telemetry_stable.main_v4 will become
-# the source for main_summary, etc. once we are comfortable retiring parquet
-# data imports.
-copy_deduplicate_main_ping = bigquery_etl_copy_deduplicate(
-    task_id="copy_deduplicate_main_ping",
-    target_project_id="moz-fx-data-shared-prod",
-    only_tables=["telemetry_live.main_v4"],
-    parallelism=24,
-    slices=100,
-    owner="jklukas@mozilla.com",
-    email=["telemetry-alerts@mozilla.com", "relud@mozilla.com", "jklukas@mozilla.com"],
-    dag=dag)
-
-bq_main_events = bigquery_etl_query(
-    task_id="bq_main_events",
-    project_id="moz-fx-data-shared-prod",
-    destination_table="main_events_v1",
-    dataset_id="telemetry_derived",
-    owner="ssuh@mozilla.com",
-    email=["telemetry-alerts@mozilla.com", "ssuh@mozilla.com"],
-    dag=dag,
-    arguments=('--schema_update_option=ALLOW_FIELD_ADDITION',),
-)
-
-main_summary = bigquery_etl_query(
-    task_id="main_summary",
-    destination_table="main_summary_v4",
-    project_id="moz-fx-data-shared-prod",
-    dataset_id="telemetry_derived",
-    sql_file_path="sql/telemetry_derived/main_summary_v4/",
-    multipart=True,
-    owner="relud@mozilla.com",
-    email=["telemetry-alerts@mozilla.com", "relud@mozilla.com"],
-    start_date=datetime(2019, 10, 25),
-    dag=dag)
+dag = DAG('parquet_export', default_args=default_args, schedule_interval='0 1 * * *', max_active_runs=10)
```

Comment:
```suggestion
dag = DAG('parquet_export', default_args=default_args, schedule_interval='0 1 * * *')
```

I assume max_active_runs was an optimization for main_summary. We should definitely get @haroldwoo's opinion on all of this before merging. He has implemented some main_summary-specific configuration due to the criticality of that workload; I believe we have a separate task queue or slot allotment for main_summary.

Now that we're splitting this ETL to multiple DAGs, we may want to remove that main_summary-specific configuration. Or perhaps extend it to the bqetl_main_summary and bqetl_clients_daily DAGs.

Or, there may be an argument for keeping the whole main_summary through clients_daily and exact_mau tasks in one DAG due to the ability to reserve task capacity for that pipeline.